sort.rb

```diff
@@ -0,0 +1,32 @@
+def bubble_sort(arr)
+  loops = arr.length
+  i = 0
+  while i < loops - 1
+    j = 0
+    while j < loops - 1
+      arr[j], arr[j + 1] = arr[j + 1], arr[j] if arr[j] > arr[j + 1]
+      j += 1
+    end
+    i += 1
+  end
+  puts arr
```

Comment:
The task of the sort is to return a sorted array. But I can see puts. Please remove `puts` and it should be only `arr`.